### ISSUE TITLE
`bijectors.Tanh` produces values strictly larger than 1 and smaller than -1

### ISSUE THREAD

**Issue by hartikainen** [tensorflow/probability#664]
I'm using the Tanh bijector and noticed that sometimes the output values become strictly larger than 1 or smaller than -1. Is this expected?

For example:
```python
import numpy as np
import tensorflow as tf
import tensorflow_probability as tfp

tf.enable_eager_execution()

x = np.float(-8.51089)
y = tfp.bijectors.Tanh()(x)

assert np.all(np.greater_equal(y, -1.0)), y
```

outputs:
```
Traceback (most recent call last):
  File "./tests/tfp_tanh_overflow_test.py", line 11, in <module>
    assert np.all(np.greater_equal(y, -1.0)), y
AssertionError: tf.Tensor(-1.0000001, shape=(), dtype=float32)
```

This happens with the following versions:
```
$ pip freeze | grep "tf\|tensor"
tensorboard==1.15.0
tensorflow==1.15.0
tensorflow-estimator==1.15.1
tensorflow-probability==0.8.0
```

**Comment by axch** [comment]
With apologies to Master Yoda, unfortunate this is, if not unexpected.  The mistake is propagated from the underlying TensorFlow tanh function, which I have filed as a separate issue: https://github.com/tensorflow/tensorflow/issues/34773.

If this is getting in your way, you could work in 64 bits, or maybe write your own variant of the Tanh bijector that clips the values?

**Comment by axch** [comment]
This has been fixed upstream.  The fix should be available in TF nightlies as of version 2.1.0-dev20191226, and will make it into the TF 2.1.0 release.

**Comment by hartikainen** [comment]
Thanks for the quick fix, @axch!
